cluster.yml

```diff
@@ -0,0 +1,49 @@
+version: "2.0"
+services:
+  choralweb:
+    build: .
+    command: npm run-script dev
```

Comment:
We should have a npm run script cluster script that runs a different command. Nodemon is just for development